common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3858,7 +3858,7 @@ object CelebornConf extends Logging {
       .doc("If direct memory usage is less than this limit, worker will resume.")
       .version("0.2.0")
       .doubleConf
-      .createWithDefault(0.7)
+      .createWithDefault(0.3)
```

Comment:
Maybe we can add a new conf  for pinnedMemoryToResume  and keep exist conf for directMemoryRatioToResume